src/CommandingOfficers/Tech.java

```diff
@@ -326,34 +314,13 @@ private TeleportEvent generateDropEvent(MapMaster gameMap, Set<XYCoord> priorDro
         // Assign base scores for spaces around this enemy.
         for( XYCoord xyc : Utils.findLocationsInRange(gameMap, nmexy, 0, dropRange) )
         {
```

Comment:
Could add an `if(invalidDropCoords.contains(xyc)) continue;` here to skip some processing.